screens/AddUserToChatScreen.js

```diff
@@ -0,0 +1,220 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import {
+    Platform,
+    ScrollView,
+    StyleSheet,
+    TouchableOpacity,
+    View,
+    KeyboardAvoidingView,
+    AsyncStorage
+} from 'react-native';
+import {
+    Container,
+    Header,
+    Title,
+    Content,
+    Form,
+    Label,
+    Button,
+    Left,
+    Right,
+    Body,
+    Icon,
+    Text,
+    List,
+    ListItem,
+    Item,
+    Input,
+    Toast
+} from 'native-base';
+
+import { baseUrl } from '../constants/api';
+
+import { putUserInChat } from '../actions/chats';
+import {
+    handleResponse,
+    addAuthHeader
+} from '../helpers/api';
+
+const AddUserForm = ({ addUserToChat, cid: chatId, user }) => {
+    const { 
+        id: userId, 
+        display_name: displayName
+    } = user;
+    return (
+        <Form style={styles.contentContainer}>
+            <Item stackedLabel>
+                <Label>User display name</Label>
+                <Text>
+                    { displayName }
+                </Text>
+            </Item>
+            <Button 
+                full
+                onPress={() => addUserToChat(chatId, userId)}>
+                <Text>Add To Chat</Text>
+            </Button>
+        </Form>
+    );
+}
+
+class AddUserToChatScreen extends React.Component {
+    static navigationOptions = {
+        header: null
+    };
+
+    state = {
+        email: '',
+        foundUser: null,
+        error: ''
+    };
+
+    _submitForm = async () => {
+        const { email } = this.state;
+        const headers = await addAuthHeader();
+        return fetch(`${baseUrl}user/${email}`, {
+            headers
+        })
+            .then(handleResponse)
+            .then((data) => {
+                this.setState({
+                    foundUser: data,
+                    email: ''
+                });
+            })
+            .catch(({ message }) => {
+                this.setState({
+                    error: message
+                });
+            });
+    };
+
+    componentDidUpdate(prevProps, prevState, snapshot) {
```

Comment:
this is a minor thing but you don't have to pass prevState and snapshot since you're not using them